app/scripts/views/settings/emails.js

```diff
@@ -0,0 +1,139 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+define(function (require, exports, module) {
+  'use strict';
+
+  const $ = require('jquery');
+  const AuthErrors = require('lib/auth-errors');
+  const BaseView = require('views/base');
+  const Cocktail = require('cocktail');
+  const Email = require('models/email');
+  const FloatingPlaceholderMixin = require('views/mixins/floating-placeholder-mixin');
+  const FormView = require('views/form');
+  const preventDefaultThen = require('views/base').preventDefaultThen;
+  const SettingsPanelMixin = require('views/mixins/settings-panel-mixin');
+  const showProgressIndicator = require('views/decorators/progress_indicator');
+  const Template = require('stache!templates/settings/emails');
+
+  var t = BaseView.t;
+
+  const EMAIL_INPUT_SELECTOR = 'input.new-email';
+  const EMAIL_REFRESH_SELECTOR = 'button.settings-button.email-refresh';
+  const EMAIL_REFRESH_DELAYMS = 350;
```

Comment:
from devices and apps @ryanfeeley wanted this to be 1600, can we move this to Constants.js?